identity/Cargo.toml

```diff
@@ -34,14 +34,17 @@ default = ["async"]
 async = ["identity-iota/async"]
 
 # Enables Web Assembly support
-wasm = ["identity-iota/wasm", "identity-comm/wasm", "identity-core/wasm"]
+wasm = ["identity-iota/wasm", "identity-comm/wasm", "identity-core/wasm", "identity-account/wasm"]
 
 # Enables support for secure storage of DID Documents
-account = ["identity-account"]
+account = ["identity-account", "wasm"]
```

Comment:
I'll repeat the comment from @HenriqueNogara for visibility: `wasm` should not be added here.